src/uct/cuda/cuda_copy/cuda_copy_ep.c

```diff
@@ -126,6 +225,7 @@ uct_cuda_copy_post_cuda_async_copy(uct_ep_h tl_ep, void *dst, void *src,
 {
     uct_cuda_copy_iface_t *iface = ucs_derived_of(tl_ep->iface, uct_cuda_copy_iface_t);
     uct_base_iface_t *base_iface = ucs_derived_of(tl_ep->iface, uct_base_iface_t);
+    CUdevice device;
```

Comment:
let's align the code so CUdevice variables /params are named "cuda_device" everywhere.
for example, sometimes the parameter is called "CUdevice *cuda_device_p"